src/libs/Environment/getEnvironment/index.native.js

```diff
@@ -1,7 +1,7 @@
 import lodashGet from 'lodash/get';
 import Config from 'react-native-config';
-import betaChecker from './betaChecker';
-import CONST from '../../CONST';
+import betaChecker from '../betaChecker/index';
```

Comment:
```suggestion
import betaChecker from '../betaChecker';
```
No need for that. The modules are smart enough to figure it out